9.secretstore/README.md

```diff
@@ -0,0 +1,144 @@
+# Secrets API
+
+This tutorial will help you write a node app to access secrets stored in a Secret Store using the Dapr Secrets API.
+
+
+
+## Prerequisites
+This sample requires you to have the following installed on your machine:
+- [Docker](https://docs.docker.com/)
+- [Node.js version 8 or greater](https://nodejs.org/en/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- A Kubernetes cluster, such as [Minikube](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#setup-cluster), [AKS](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#setup-cluster) or [GKE](https://cloud.google.com/kubernetes-engine/)
+- [Postman](https://www.getpostman.com/) [Optional]
+
+Also, unless you have already done so, clone the repository with the samples and ````cd```` into the right directory:
+```
+git clone https://github.com/dapr/samples.git
+cd samples
+```
+  
+## Step 1 - Setup Dapr on your Kubernetes Cluster
+
+The first thing you need is an RBAC enabled Kubernetes cluster. This could be running on your machine using Minikube, or it could be a fully-fledged cluser in Azure using [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/). 
+
+Once you have a cluster, follow the steps below to deploy Dapr to it. For more details, look [here](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#installing-dapr-on-a-kubernetes-cluster)
+
+> Please note, that using the CLI does not support non-default namespaces.  
+> If you need a non-default namespace, Helm has to be used (see below).
+
+```
+$ dapr init --kubernetes
+ℹ️  Note: this installation is recommended for testing purposes. For production environments, please use Helm
+
+⌛  Making the jump to hyperspace...
+✅  Deploying the Dapr Operator to your cluster...
+✅  Success! Dapr has been installed. To verify, run 'kubectl get pods -w' in your terminal
+```
+
+## Step 2 - Configure a Secret in the Secret Store
+
+Dapr can use a number of different secret stores (AWS Secret Manager, Azure Key Vault, GCP Secret Manager, Kubernetes, etc) to retrieve secrets. For this demo, we'll use Kubernetes. For other secret stores, you can follow the instructions at https://github.com/dapr/docs/tree/master/howto/setup-secret-store
+
+1. Add your secret to a file ./secret
+2. Create the secret in the Kubernetes secret store. Note, the name of our secret here is "mysecret" and will be used in a later step.
+    ```
+    kubectl create secret generic mysecret --from-file ./secret
+    ```
+3. You can check that the secret is indeed stored in the Kubernetes secret store by running the command:
+    ```
+    kubectl get secret mysecret -o yaml
+    ```
+   You can see the output as below where the secret is stored in the secret store in Base64 encoded format
+   ```
+    % k get secret mysecret -o yaml
```

Comment:
k is a frequently used alias, but it should be spelled out here.

Reply:
fixed